ci-operator/step-registry/ipi/install/monitoringpvc/ipi-install-monitoringpvc-commands.sh

```diff
@@ -23,3 +32,5 @@ data:
             requests:
               storage: 10Gi
 EOF
+
+/tmp/yq m -a -i "${CONFIG}" "${PATCH}"
```

Comment:
the `config.yaml` property is a string, right?  Is `yq` smart enough to say "ah, the body of this string is also YAML, let me parse as YAML and add the new keys..."?

Reply:
Yup, it merges keys as YAML. Let me add a step to setup PVC for these jobs too
```

Reply:
Ah, I was wrong, we have to extract it ourselves:
* [UMW + PVC](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/19545/rehearse-19545-periodic-ci-openshift-release-master-ci-4.7-upgrade-from-stable-4.6-e2e-aws-umw-upgrade/1408361709512429568): [initial config](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/19545/rehearse-19545-periodic-ci-openshift-release-master-ci-4.7-upgrade-from-stable-4.6-e2e-aws-umw-upgrade/1408361709512429568/artifacts/e2e-aws-umw-upgrade/ipi-conf-user-workload-monitoring/build-log.txt) and [final](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/19545/rehearse-19545-periodic-ci-openshift-release-master-ci-4.7-upgrade-from-stable-4.6-e2e-aws-umw-upgrade/1408361709512429568/artifacts/e2e-aws-umw-upgrade/ipi-install-monitoringpvc/build-log.txt)
* [PVC only](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/19545/rehearse-19545-periodic-ci-openshift-release-master-ci-4.9-upgrade-from-stable-4.8-e2e-gcp-ovn-upgrade/1408361720421814272): [config](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/19545/rehearse-19545-periodic-ci-openshift-release-master-ci-4.9-upgrade-from-stable-4.8-e2e-gcp-ovn-upgrade/1408361720421814272/artifacts/e2e-gcp-ovn-upgrade/ipi-install-monitoringpvc/build-log.txt)